core/webservice_default.go

```diff
@@ -23,12 +23,12 @@ func (i *InfoCMDB) QueryWebservice(ws string, params map[string]string) (r strin
 	}
 
 	var res interface{}
-	resp, err := i.v2.Query(ws, &res, params)
+	err = i.v2.Query(ws, &res, params)
 	if err != nil {
 		log.Error("Error: ", err)
 		return "", err
 	}
-	r = resp.String()
+
```

Comment:
This func requires the return of the raw json for custom processing. please fix this.